engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -118,6 +118,8 @@
   public static final long DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
   public static final String TOKEN_REFRESH_PRINCIPAL = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.principal";
   public static final String TOKEN_REFRESH_KEYTAB = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.keytab";
+  public static final String CUSTOM_SSL_KEYSTORE_CONFIG = StreamingApplication.APEX_PREFIX + "ssl.keystore.config";
```

Comment:
It was @tweise 's suggestion to use apex instead of dt so I would like to get his response. Also @vrozov the last line in your 1st para says "...this PR should use "dt" prefix ..." . Did you mean to say "...this PR should _also_ use "dt" prefix..." .